src/main/java/com/openshift/restclient/model/volume/IVolumeMount.java

```diff
@@ -0,0 +1,21 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package com.openshift.restclient.model.volume;
+
+/**
+ * The volumes that are mounted 
+ * in a container
+ * @author jeff.cantrill
+ *
+ */
```

Comment:
Extended the deprecated class with the thought of removing it all together to the correct concept.  I'm hoping to refactor to remove the dependency